spec/features/users/sign_up_spec.rb

```diff
@@ -39,7 +39,8 @@
 
     it 'allows a user to pick a language when entering email' do
       visit sign_up_email_path
-      fill_in 'Email', with: email
+      find('#user_terms_accepted').set(true)
```

Comment:
I think we can use the Capybara `check` helper here:

```suggestion
      check '#user_terms_accepted'
```

Reply:
will break the js specs

Reply:
Can we have them click by title instead?
```suggestion
      check t('account.terms')
```

Reply:
I checked out the branch locally and `check t('account.terms')` works

Reply:
That worked...NICE!!!

Reply:
Broken again.  I needed to make UI changes per Annie and that one js spec is broken again.